behaviour/graql/language/match.feature

```diff
@@ -770,6 +914,19 @@ Feature: Graql Match Clause
       | NIN |
 
 
+  Scenario: when querying for a non-existent attribute type id, an empty result is returned
```

Comment:
have we got a test to check that retriving an attribtue by value that doesn't exist throws or is a noop?

Reply:
Added [Scenario Outline: when matching a `<type>` attribute by a value that doesn't exist, an empty answer is returned]